docs/development/creating-editing-renovate-presets.md

```diff
@@ -0,0 +1,38 @@
+# Creating/editing Renovate presets
+
+Renovate comes with default presets that you can find in the `lib/config/presets/internal` directory.
+You can suggest changes to the presets with a pull request.
+
+Follow the rules below to increase the chance that your pull request gets merged.
+
+## General rules
+
+1. Only create/edit presets for problems which can not be fixed upstream
```

Comment:
I think we should nudge our users/contributors into fixing the problem upstream, that way bad stuff gets fixed for all downstream users not just Renovate bot itself.

It also means less maintenance work for us. 😄